server/common/utils/helpers.test.js

```diff
@@ -0,0 +1,391 @@
+const { ObjectId } = require('mongoose').Types;
+
+const {
+  checkIfArrayContainsUserId,
+  checkIfCohortMember,
+  checkIfCurrentUserVoted,
+  checkIfGuest,
+  isUserFavourite,
+  isValidMongoId,
+  responseWithCohort,
+  responseWithCohortMember,
+  responseWithCohortMembers,
+  responseWithCohorts,
+  responseWithItem,
+  responseWithItems,
+  responseWithList,
+  responseWithListMember,
+  responseWithListMembers,
+  responseWithListsMetaData
+} = require('./index');
+const {
+  expectedCohortListProperties,
+  expectedListMetaDataProperties,
+  expectedListProperties,
+  listMock
+} = require('../../tests/__mocks__/listMock');
+const { expectedItemProperties } = require('../../tests/__mocks__/itemsMock');
+const { singleItemMock } = require('../../tests/__mocks__/singleItemMock');
+const {
+  cohortsMock,
+  expectedCohortMetaDataProperties
+} = require('../../tests/__mocks__/cohortsMock');
+const {
+  expectedUsersProperties,
+  usersMock
+} = require('../../tests/__mocks__/usersMock');
+
+describe('function isValidMongoId', () => {
+  it('checks if passed ID is a valid mongo id', () => {
+    const id = ObjectId();
+    const result = isValidMongoId(id);
+
+    expect(id).toBeInstanceOf(ObjectId);
+    expect(result).toBe(true);
+  });
+
+  it('check if passed ID is an invalid mongo id', () => {
```

Comment:
should be named `returns false if passed ID is a an invalid mongo id`. Do we even need this function? Can't we just use the ObjectId.isValid(id)?